packages/relay/src/lib/eth.ts

```diff
@@ -1262,15 +1268,16 @@ export class EthImpl implements Eth {
         }
 
         if (e.isContractReverted()) {
-          this.logger.trace(`${requestIdPrefix} mirror node eth_call request encoutered contract revert. message: ${e.message}, details: ${e.detail}, data: ${e.data}`);
-          return predefined.CONTRACT_REVERT(e.detail || e.message, e.data);
+          this.logger.trace(`${requestIdPrefix} mirror node eth_call request encoutered contract revert. details: ${e.detail}, data: ${e.data}`);
+          return predefined.CONTRACT_REVERT(e.message + e.detail, e.data);
```

Comment:
Q: why this change?
Is this in the case both message and detail are populated?

Reply:
Should not be something like this preferable:
```suggestion
          return predefined.CONTRACT_REVERT(`message: ${e.message}, detail: ${e.detail}`, e.data);
```
or
```suggestion
          return predefined.CONTRACT_REVERT(`${e.message} : ${e.detail}`, e.data);
```